datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/converter/GrpcStatusResponseToDatalakeBackupRestoreStatusResponseConverter.java

```diff
@@ -18,34 +20,56 @@ public class GrpcStatusResponseToDatalakeBackupRestoreStatusResponseConverter {
 
     static final String FAILED_STATE = "FAILED";
 
+    static final String VALIDATION_FAILED_STATE = "VALIDATION_FAILED";
+
+    static final String SUCCESSFUL_STATUS = "SUCCESSFUL";
+
+    private static final Set<String> FAILED_STATES = Set.of(FAILED_STATE, VALIDATION_FAILED_STATE);
+
     private static final Logger LOGGER = LoggerFactory.getLogger(GrpcStatusResponseToDatalakeBackupRestoreStatusResponseConverter.class);
 
     public DatalakeBackupStatusResponse convert(datalakeDRProto.BackupDatalakeResponse response) {
         return new DatalakeBackupStatusResponse(response.getBackupId(),
                 DatalakeBackupStatusResponse.State.valueOf(response.getOverallState()),
-                Optional.ofNullable(parseFailuresFromOperationsStates(response.getOperationStates(), response.getFailureReason()))
-        );
+                List.of(), "", parseFailuresFromOperationsStates(response.getOperationStates(), response.getFailureReason()));
     }
 
     public DatalakeRestoreStatusResponse convert(datalakeDRProto.RestoreDatalakeResponse response) {
         return new DatalakeRestoreStatusResponse(response.getBackupId(), response.getRestoreId(),
                 DatalakeRestoreStatusResponse.State.valueOf(response.getOverallState()),
-                Optional.ofNullable(parseFailuresFromOperationsStates(response.getOperationStates(), response.getFailureReason()))
+                parseFailuresFromOperationsStates(response.getOperationStates(), response.getFailureReason())
         );
     }
 
     public DatalakeBackupStatusResponse convert(datalakeDRProto.BackupDatalakeStatusResponse response) {
+        List<String> contents = new LinkedList<>();
+        datalakeDRProto.InternalBackupRestoreState operationStates = response.getOperationStates();
+        datalakeDRProto.SolrBackupRestoreState solrState = operationStates.getSolr();
+
+        if (solrState.getRangerAuditsCollection().getStatus().equals(SUCCESSFUL_STATUS)) {
+            contents.add("RANGER_AUDITS");
+        }
+        if (operationStates.getDatabase().getDatabase().getStatus().equals(SUCCESSFUL_STATUS)) {
+            contents.add("RANGER_PERMISSIONS");
+            contents.add("HMS_METADATA");
+        }
+        if (solrState.getEdgeIndexCollection().getStatus().equals(SUCCESSFUL_STATUS)
+                && solrState.getFulltextIndexCollection().getStatus().equals(SUCCESSFUL_STATUS)
+                && solrState.getVertexIndexCollection().getStatus().equals(SUCCESSFUL_STATUS)
+                && solrState.getEdgeIndexCollection().getStatus().equals(SUCCESSFUL_STATUS)) {
+            contents.add("ATLAS_METADATA");
```

Comment:
@votez @kkalvagadda1 
`ATLAS_METADATA`, Is It enough? 
Why we do not want to give all of these if we can? 
e.q.:
`EDGE INDEX`
`VERTEX INDEX`
....

Reply:
according to the Jira ticket the combination of these belongs to Atlas so I only report Atlas not to confuse a client with fine granularity. Would you suggest to add more detailed list of data backed up?

Reply:
I just asking.
I can accept this.  So I resolved this conversation. Thank you!